python/gtsam/tests/test_DiscreteBayesNet.py

```diff
@@ -53,24 +53,18 @@ def test_Asia(self):
         XRay = (2, 2)
         Dyspnea = (1, 2)
 
-        def P(keys):
-            dks = DiscreteKeys()
-            for key in keys:
-                dks.push_back(key)
-            return dks
-
         asia = DiscreteBayesNet()
-        asia.add(Asia, P([]), "99/1")
-        asia.add(Smoking, P([]), "50/50")
+        asia.add(Asia, "99/1")
+        asia.add(Smoking, "50/50")
 
-        asia.add(Tuberculosis, P([Asia]), "99/1 95/5")
-        asia.add(LungCancer, P([Smoking]), "99/1 90/10")
-        asia.add(Bronchitis, P([Smoking]), "70/30 40/60")
+        asia.add(Tuberculosis, "99/1 95/5", Asia)
+        asia.add(LungCancer, "99/1 90/10", Smoking)
+        asia.add(Bronchitis, "70/30 40/60", Smoking)
 
-        asia.add(Either, P([Tuberculosis, LungCancer]), "F T T T")
+        asia.add(Either, "F T T T", Tuberculosis, LungCancer)
```

Comment:
Test the constructor with array of tuples?

Reply:
Done